packages/common/src/utils.js

```diff
@@ -480,3 +480,15 @@ export function serializeFunction(func) {
 
 serializeFunction.internalFunctionRE = /^(\s*)(?!(?:if)|(?:for)|(?:while)|(?:switch))(\w+)\s*\((.*?)\)\s*\{/gm
 serializeFunction.assignmentRE = /^(\s*):(\w+)\(/gm
+
+export function tryRequire(pkg) {
```

Comment:
We have this util in nuxt.resolver

Reply:
@pi0 `requireModule` from `nuxt.resolver` throws an error if the package is not found.
I wanted to find a way to continue the process with a warning instead of making the build fail. What do you suggest then ?

Reply:
use `resolveModule` in `resolver`